tests/lib/rules/no-inline-comments.js

```diff
@@ -88,7 +88,24 @@ ruleTester.run("no-inline-comments", rule, {
                comment
             */}
             </div>
-        )`
+        )`,
+
+        // Webpack
+        {
+            code: `
+              import(
+                /* webpackInclude: /\\.json$/ */
+                /* webpackChunkName: "my-chunk-name" */
+                './locale/en'
```

Comment:
This example would be valid even without the pattern, since the comments are on separate lines?

Reply:
Now that's silly 🤦‍♀️